### PR TITLE
Updated Wallet documentation for RC73.

### PR DESCRIPTION
**Pull request by ingerjm0**
Removed obsolete Wallet documentation and added a FAQ about old behavior.

### CODE REVIEW

File: 07.high-fidelity-commerce/01.basics/01.hfc/02.send-money/docs.md [hunk 1]
```diff
@@ -12,10 +12,9 @@ This page details how you can transfer HFC from your wallet to another user.
 
 ## Introduction
 
-Once you are done with setting up your wallet and receiving your HFC, learn how to send HFC to another user. 
+Once you have logged into High Fidelity and [received your HFC](http://localhost:8000/high-fidelity-commerce/basics/hfc/get-hfc), learn how to send HFC to another user.
```
Comment (zfox23) [inline 07.high-fidelity-commerce/01.basics/01.hfc/02.send-money/docs.md]: Will this link get updated when this PR is pushed live?

File: 07.high-fidelity-commerce/01.basics/01.hfc/docs.md [hunk 2]
```diff
@@ -10,11 +10,9 @@ taxonomy:
 
 High Fidelity Coins is a cryptocurrency that is empowered by blockchain technology.
 
-We are currently giving out the currency for anyone interested in participating in the closed beta for High Fidelity Commerce. If you want to [get your inital HFC grant](/high-fidelity-commerce/basics/hfc/get-hfc), you first need to [set up your Wallet](https://docs.highfidelity.com/high-fidelity-commerce/basics/hfc/wallet-setup).
+We are currently giving out the currency for anyone interested in participating in the closed beta for High Fidelity Commerce. These coins are to be used as currency for any commerce transactions in the Marketplace. Since we are using blockchain technology, all transactions with HFCs will be publically recorded and stored.
```
Comment (zfox23) [inline 07.high-fidelity-commerce/01.basics/01.hfc/docs.md]: Looks like "publicly" is more widely accepted than "publically". Does it matter?
http://www.macmillandictionaryblog.com/is-there-a-case-for-publically-part-1

File: 07.high-fidelity-commerce/01.basics/01.hfc/docs.md [hunk 3]
```diff
@@ -10,11 +10,9 @@ taxonomy:
...
+We are currently giving out the currency for anyone interested in participating in the closed beta for High Fidelity Commerce. These coins are to be used as currency for any commerce transactions in the Marketplace. Since we are using blockchain technology, all transactions with HFCs will be publically recorded and stored. 
 
-These coins are to be used as currency for any commerce transactions in the Marketplace. Since we are using blockchain technology, all transactions with HFCs will be publically recorded and stored. 
-
-Your Wallet will be secured using ECDSA public-private key pairs, security picture and passphrase. Learn more about your Wallet [here](https://docs.highfidelity.com/high-fidelity-commerce/basics/hfc/wallet-setup). 
+Your Wallet is accessed by logging into your High Fidelity account. In addition, you will select a Security Picture, which serves as an extra layer of security for your Wallet. When you see your Security Picture, you know that your actions and data are securely making use of your account.
```
Comment (zfox23) [inline 07.high-fidelity-commerce/01.basics/01.hfc/docs.md]: Users are now assigned a security pic; they aren't asked to select one. They _can_ change their randomly-assigned security pic in Settings, but for now, this isn't correct.

File: 07.high-fidelity-commerce/01.basics/docs.md [hunk 4]
```diff
@@ -12,10 +12,8 @@ Here is some basic information that will help you understand how commerce works
...
 
-* **Cryptographically-secured Wallet**: Users will be able to participate in transactions using their Wallet, which will be an app on their tablet in High Fidelity. Your Wallet is secured using a security picture and a passphrase which includes ECDSA public-private keys pairs. These key pairs are used to sign each transaction. 
+* **Cryptographically-secured Wallet**: Users will be able to participate in transactions using their Wallet, which will be an app on their tablet in High Fidelity. Your Wallet is accessed by logging into your High Fidelity account, and is additionally secured using a security picture.
```
Comment (zfox23) [inline 07.high-fidelity-commerce/01.basics/docs.md]: "secured using" -> "secured by" ?

File: 07.high-fidelity-commerce/03.avatar-wearables/01.buy/docs.md [hunk 5]
```diff
@@ -28,7 +28,7 @@ Avatar Island is a High Fidelity domain where you can purchase wearables created
 
   > > > > > If you don't want to purchase any of the wearables, you can simply walk out of the store without checking out. All wearables you are wearing will disappear when you leave. You can also take them off in VR mode or delete them using the **Wear** app in Desktop mode. If you want to see how your avatar looks with various wearables on, you can look at yourself in one of the many mirrors in the store. 
 
-- Once you are done selecting the wearables you want to buy, head over to one of the checkout kiosks. When you are close to a kiosk, you will see the items you picked in the shelf marked "Pick Items". Your tablet will also open your wallet and ask for your passphrase. 
+- Once you are done selecting the wearables you want to buy, head over to one of the checkout kiosks. When you are close to a kiosk, you will see the items you picked in the shelf marked "Pick Items". Your tablet will also open your wallet.
```
Comment (zfox23) [inline 07.high-fidelity-commerce/03.avatar-wearables/01.buy/docs.md]: Users can also purchase wearables straight from Marketplace (in addition to being able to try them on in Avatar Island). Is that something we want to note here?

File: 07.high-fidelity-commerce/01.basics/01.hfc/docs.md (hunk 2 above)
Comment (ingerjm0) [reply 07.high-fidelity-commerce/01.basics/01.hfc/docs.md]: I agree with you...this was existing documentation. I'll go ahead and change it.

File: 07.high-fidelity-commerce/03.avatar-wearables/01.buy/docs.md (hunk 5 above)
Comment (ingerjm0) [reply 07.high-fidelity-commerce/03.avatar-wearables/01.buy/docs.md]: Added note saying such.

File: 07.high-fidelity-commerce/01.basics/01.hfc/02.send-money/docs.md (hunk 1 above)
Comment (ingerjm0) [reply 07.high-fidelity-commerce/01.basics/01.hfc/02.send-money/docs.md]: Fixed

File: 07.high-fidelity-commerce/01.basics/01.hfc/docs.md (hunk 3 above)
Comment (ingerjm0) [reply 07.high-fidelity-commerce/01.basics/01.hfc/docs.md]: Fixed

File: 07.high-fidelity-commerce/01.basics/docs.md (hunk 4 above)
Comment (ingerjm0) [reply 07.high-fidelity-commerce/01.basics/docs.md]: Fixed

**Pull request merged**